bin/tectonics-modeling-L0/session1-aspect.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    ax.set_xlabel("Time (years)")
</pre><p>I think it would be nice to add the y axis label here and above too:</p><p><code>ax.set_ylabel("Velocity (m/year)")</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>